NOTICE.md

```diff
@@ -29,21 +29,27 @@ in the GitHub organization https://github.com/eclipse-tractusx:
 
 * https://github.com/eclipse-tractusx/puris
 
-
 ## Third-party Content
 
 This project leverages the following third party content.
 
 See `DEPENDENCIES_FRONTEND` and `DEPENDENCIES_BACKEND` file. Further, the following third-party content is used that
 isn't listed in any DEPENDENCIES file:
 
-Keycloak initial realm setup
+[Keycloak initial realm setup](./local/miw/keycloak-setup.json)
```

Comment:
would be beneficial to replace this in the future with actual right&roles concept which we maintain https://github.com/eclipse-tractusx/portal-iam/blob/v3.0.0-rc.3/import/realm-config/generic/catenax-central/CX-Central-realm.json

Reply:
Converted comment to issue!